### ISSUE TITLE
Compilation creates lots of warnings

### ISSUE THREAD

**Issue by datee** [RobertBorghese/Haxe-UnrealEngine5#19]
So, compiling a project seems to generate lots and lots of hxcpp warnings.. any idea whats going on here?
Even so, the project complies and runs ok - but it would definitly be nice to remove or at least supress these?

(I've attached a file with the warnings)
[warnings.txt](https://github.com/RobertBorghese/Haxe-UnrealEngine5/files/9895664/warnings.txt)

**Comment by SomeRanDev** [comment]
Yes, this is normal, you can see my comments about it here: https://github.com/RobertBorghese/Haxe-UnrealEngine5/issues/3#issuecomment-1264933322

As mentioned there, unfortunately all of the warnings come from hxcpp, a required library for compiling Haxe/C++ output, but one that does not have many errors enabled in its compilation environment.

It’s possible to suppress these errors, but it’d be kind of a hassle, so haven’t gotten around to it yet. Afaik, Unreal doesn’t provide any system for suppressing any warnings/errors, just a select handful of ones. So the only solution is to clutter up the output files with tons of C++ suppress macro things.

As this project is still in early development, they’ll be left on for testing purposes for the time being, but open to changing this in future.

**Comment by datee** [comment]
Ah, yes. Sorry, i didnt see your earlier comment about this. Will leave this issue open still for now, in case others have the same question!
